src/requirements.txt

```diff
@@ -0,0 +1,2 @@
+obsah >= 1.1
+apypie>=0.5.0
```

Comment:
technically, we don't need apypie (it's vendored in the ansible collection), only requests for the prod parts.

apypie *is* used in tests tho.

I think that's OK for now, but automatic dependency generation will hate you if we ever enable it :)

Reply:
I'll adjust it, I didn't realize that part of it.